site/sfguides/src/data_collaboration_native_app/data_collaboration_native_app.md

```diff
@@ -0,0 +1,630 @@
+CREATE OR REPLACE STAGE CREDIT_CARD_PREDICTION_APP_PACKAGE.MODEL_ASSETS.MODEL_STAGE FILE_FORMAT = (TYPE = 'csv' FIELD_DELIMITER = '|' SKIP_HEADER = 1);
+```
+
+So we share a model that is consistent with what is in the model registry, we are going to export the model from the registry, and put it in the Application Package. We do this in the following code block
+
+```python
+# Retrieve the model from the registry
+m = registry.get_model("NATIVE_APP_DEMO.NATIVE_APP_DEMO.CREDIT_CARD_DEFAULT_MODEL")
+mv = m.version("default")
+
+# Create a temp directory to export the model
+import tempfile, pathlib, shutil, os
+local_dir = pathlib.Path(tempfile.mkdtemp(prefix="model_export_"))
+
+# export the full artefact set into that folder
+mv.export(local_dir.as_posix(), export_mode=ExportMode.FULL)
+
+# Upload the files and folder structure to the Application Package Stage
+stage_root = "@CREDIT_CARD_PREDICTION_PACKAGE.MODEL_ASSETS.MODEL_STAGE/models"
+
+for file_path in pathlib.Path(local_dir).rglob('*'):
+    if file_path.is_file():
+        rel_path = file_path.relative_to(local_dir).as_posix()   # e.g. runtimes/env.yml or manifest.yml
+        subdir   = os.path.dirname(rel_path)                     # '' or 'runtimes'
+
+        # remote prefix **must end with a slash** and contain no file-name
+        remote_prefix = f"{stage_root}/{subdir}/" if subdir else f"{stage_root}/"
+
+        session.file.put(
+            file_path.as_posix(),
+            remote_prefix,
+            overwrite=True,
+            auto_compress=False
+        )
+
+# Check that all our artefacts have been uploaded
+session.sql("LIST @CREDIT_CARD_PREDICTION_PACKAGE.MODEL_ASSETS.MODEL_STAGE/models;").collect()
+```
+
+An application requires a manifest file, and a setup script (as outlined in the Native App Development Workflow). We upload them to our Application Package in the following commands
+
+```SQL
+PUT file://scripts/setup.sql @CREDIT_CARD_PREDICTION_APP_PACKAGE.MODEL_ASSETS.MODEL_STAGE/scripts overwrite=true auto_compress=false;
+
+PUT file://manifest.yml @CREDIT_CARD_PREDICTION_APP_PACKAGE.MODEL_ASSETS.MODEL_STAGE overwrite=true auto_compress=false;
+```
+
+The setup script contains the SQL statements that define the components created when a consumer installs your application. For us, this will include creating stored procedures to perform feature engineering, as well as registering our model for the app to use on the consumer side (documentation [here](https://docs.snowflake.com/LIMITEDACCESS/native-apps/models)). This is done in the following lines that were run as part of setup.sql on installation of the app:
+
+```SQL
+create or replace model APP_CODE.CREDIT_CARD_DEFAULT_MODEL from @CREDIT_CARD_PREDICTION_PACKAGE.MODEL_ASSETS.MODEL_STAGE/models/;
+
+grant usage on model APP_CODE.CREDIT_CARD_DEFAULT_MODEL to application role app_user;
+```
+
+
+
+<!-- ------------------------ -->
+## Provider Account (Zamboni) - Perform Local Testing
```

Comment:
@sfc-gh-tbuchhorn Rename this to "Perform Local Testing" and add "Provider Account (Zamboni)" sub-header underneath.